src/gl/canvas2dRe.re

```diff
@@ -0,0 +1,76 @@
+type t;
+
+/* Canvas API, following https://simon.html5.org/dump/html5-canvas-cheat-sheet.html */
+
+external save : unit = "" [@@bs.send.pipe: t];
+external restore : unit = "" [@@bs.send.pipe: t];
+
+/* Transformation */
+external scale : float => float => unit = "" [@@bs.send.pipe: t];
+external rotate : float => unit = "" [@@bs.send.pipe: t];
+external translate : float => float => unit = "" [@@bs.send.pipe: t];
+external transform : float => float => float => float => float => float => unit = "" [@@bs.send.pipe: t];
+external setTransform : float => float => float => float => float => float => unit = "" [@@bs.send.pipe: t];
```

Comment:
Yeah a signature of 6 consecutive floats is pretty gross. The labelled version isn't really significantly easier to read though. I haven't used transform/setTransform directly myself, so I'm not aware of a cleaner solution